Automattic/wp-calypso
# Checkout: Replace all remaining blue colors with Color Studio version

## Proposed Changes

This is a follow-up to https://github.com/Automattic/wp-calypso/pull/94147 which updated the checkout blue button colors to use the "WordPress Blue" color family. In this PR we also update all the other blue colors used in the `@automattic/composite-checkout` package and other parts of checkout to also use shades of "WordPress Blue".

Before             |  After
:-------------------------:|:-------------------------:
<img width="624" alt="Screenshot 2024-09-04 at 11 57 24 AM" src="https://github.com/user-attachments/assets/6bd3ad8d-530a-46d7-a07b-dd678f1fb03f"> | <img width="621" alt="Screenshot 2024-09-04 at 11 57 31 AM" src="https://github.com/user-attachments/assets/b8f06366-b31b-4683-96fa-73d4cada2338">
<img width="302" alt="Screenshot 2024-09-04 at 11 59 50 AM" src="https://github.com/user-attachments/assets/cf601c66-85b3-4270-9998-0c0aab814fe2"> | <img width="301" alt="Screenshot 2024-09-04 at 12 18 26 PM" src="https://github.com/user-attachments/assets/8eb22a5a-4a6d-43ed-aaf6-bf2afaa6504d">
<img width="577" alt="Screenshot 2024-09-04 at 12 00 42 PM" src="https://github.com/user-attachments/assets/29ffc57c-face-4828-b3aa-26aad6133a3e"> | <img width="581" alt="Screenshot 2024-09-04 at 12 10 43 PM" src="https://github.com/user-attachments/assets/d4e027fd-59ba-4d29-9ce5-d7967b8eaa24">
<img width="581" alt="Screenshot 2024-09-04 at 12 01 32 PM" src="https://github.com/user-attachments/assets/747c2dbf-b2ac-4d82-87a1-f1441143b055"> | <img width="579" alt="Screenshot 2024-09-04 at 12 01 46 PM" src="https://github.com/user-attachments/assets/daed2c07-e232-492c-aee5-b2d2b50e7fc4">
<img width="581" alt="Screenshot 2024-09-04 at 6 39 43 PM" src="https://github.com/user-attachments/assets/b6935b3a-8d2a-4577-b767-3bdd7cfd49c3"> | <img width="574" alt="Screenshot 2024-09-04 at 6 49 44 PM" src="https://github.com/user-attachments/assets/032a8625-257c-4b95-b4c3-9ed1001f4cfe">
<img width="551" alt="Screenshot 2024-09-04 at 6 52 25 PM" src="https://github.com/user-attachments/assets/82032c6a-4ba3-455b-a757-9c8fb81286ad"> | <img width="545" alt="Screenshot 2024-09-04 at 6 52 33 PM" src="https://github.com/user-attachments/assets/44fea000-1800-4b38-8475-0f50fe34150f">
<img width="582" alt="Screenshot 2024-09-04 at 6 57 55 PM" src="https://github.com/user-attachments/assets/a9f221b2-ea60-4a8c-9710-1c0018248a99"> | <img width="575" alt="Screenshot 2024-09-04 at 6 59 05 PM" src="https://github.com/user-attachments/assets/bae105b0-4731-410d-b65a-e1557f8f7ddb">



> [!NOTE]
> This does not fix the color of the "Visit Help Center" button at the top of checkout because that is part of calypso itself and I'm actually not sure where its color comes from. It's set to `var(--wp-components-color-accent,var(--wp-admin-theme-color,#3858e9))` so I suspect that fixing it will be a calypso-wide change somewhere.

<img width="272" alt="Screenshot 2024-09-04 at 11 59 18 AM" src="https://github.com/user-attachments/assets/94986039-5a6d-49e0-aee2-0f59397d5df0">

## Testing Instructions

Go through each of the items listed in this comment and make sure they have the expected blue color: https://github.com/Automattic/wp-calypso/pull/94147#pullrequestreview-2280386090 Specifically:

- All text links should have the new blue (In terms, the support link, and edit button)
- The title for the callout in the sidebar
- The active state for the dropdown options in the term
- The borders for the payment methods (both hover and active state)
